### PR TITLE
Test parse minutes

### LINKED ISSUE #2: Parse the minutes file
Add a function

```func minutes(r io.Reader) (map[string]int, error)```

In `func main` you should pass in the `os.File` that you opened for the minutes file. Print the results.

In a test file you can use `strings.NewReader` to pass in pre-formatted versions of the minutes file.

The function itself should use the `encoding/csv` package to parse the data from the reader. The return value is a `map[string]int` where the `string` keys are the phone numbers and the `int` values are the total number of minutes for that number in the file.

### PR DESCRIPTION
**Pull request by hitjim**
Used a smidge of SO copypasta for an `indexOf`-esque generic/getter - ripe for judgement!

Closes #2 ... proberly

`go run ting-bill-split.go -minutes=minutes.csv -messages=messages.csv -megabytes=megabytes.csv`

### CODE REVIEW

File: ting-bill-split.go [hunk 1]
```diff
@@ -23,6 +37,58 @@ func readAndPrint(f *os.File) {
 	fmt.Printf("read %d bytes: %q\n", count, data[:count])
 }
 
+func parseMinutes(minReader io.Reader) map[string]int {
+	var m map[string]int
+	m = make(map[string]int)
+	r := csv.NewReader(minReader)
+
+	// Get index of important fields
+	record, err := r.Read()
```
Comment (jcbwlkr) [inline ting-bill-split.go]: I'd rename `record` to `header`

File: ting-bill-split.go [hunk 2]
```diff
@@ -62,7 +128,9 @@ func main() {
 		log.Fatal(err)
 	}
 
-	readAndPrint(minFile)
 	readAndPrint(msgFile)
 	readAndPrint(megFile)
+
+	minMap := parseMinutes(bufio.NewReader(minFile))
```
Comment (jcbwlkr) [inline ting-bill-split.go]: `minFile` is an `os.File` which already implements `io.Reader` so you don't need to pass it to `bufio`

File: ting-bill-split_test.go [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+	"testing"
+)
+
+func TestParseMinutes(t *testing.T) {
+	cases := []struct {
+		in   string
+		want map[string]int
+	}{
+		{
+			"Date,Time,Incoming/Outgoing,Phone,Nickname,Location,Country,Partner's Phone,Partner Nickname,Partner's Location,Partner's Country,Duration (min),Surcharges ($),Features\n" +
```
Comment (hitjim) [inline ting-bill-split_test.go]: I couldn't quite get that to work? I tried 
```
			`Date,Time,Incoming/Outgoing,Phone,Nickname,Location,Country,Partner's Phone,Partner Nickname,Partner's Location,Partner's Country,Duration (min),Surcharges ($),Features
				"February 03, 2011",01:11,outgoing,1112223333,Phone 1,"SPRINGFIELD, MO",USA,7778889999,,USA,United States of America,1,0.0,""
				"February 14, 2011",01:22,outgoing,1112223333,Phone 1,"SPRINGFIELD, MO",USA,7778889999,,USA,United States of America,2,0.0,""
				"February 14, 2011",01:22,outgoing,1112224444,Phone 2,"DELANO, KS",USA,7778889999,,USA,United States of America,1,0.0,""`,
```
but landed at a happy medium with this?
```
			`Date,Time,Incoming/Outgoing,Phone,Nickname,Location,Country,Partner's Phone,Partner Nickname,Partner's Location,Partner's Country,Duration (min),Surcharges ($),Features` + "\n" +
				`"February 03, 2011",01:11,outgoing,1112223333,Phone 1,"SPRINGFIELD, MO",USA,7778889999,,USA,United States of America,1,0.0,""` + "\n" +
				`"February 14, 2011",01:22,outgoing,1112223333,Phone 1,"SPRINGFIELD, MO",USA,7778889999,,USA,United States of America,2,0.0,""` + "\n" +
				`"February 14, 2011",01:22,outgoing,1112224444,Phone 2,"DELANO, KS",USA,7778889999,,USA,United States of America,1,0.0,""`,
```

**Comment by hitjim** [comment]
@jcbwlkr bloop.

**Comment by jcbwlkr** [comment]
I pushed up a few commits with the last little changes I would make. If you're good with them go ahead and click "Squash and merge"

**Pull request merged**